Sources/HTTPKit/Client/HTTPClientRequestEncoder.swift

```diff
@@ -3,18 +3,18 @@ internal final class HTTPClientRequestEncoder: ChannelOutboundHandler, Removable
     typealias OutboundIn = HTTPRequest
     typealias OutboundOut = HTTPClientRequestPart
 
-    let hostname: String
+    let host: String
```

Comment:
Maybe we should call this `hostHeader` to be extra clear it's not just the hostname?

Reply:
@tanner0101 I hesitated on this, because I was worried that including "header" in the variable name might make it feel as if the string contained a whole header in the "Host: ..." format, while in fact it's more the header _value_. I'm not used enough to server side programming to know what people would expect in this case. What do you think?

Reply:
Ah good point. `hostHeaderValue` is even better. Better to err on the side of verbosity to be clear, especially when the code is internal anyway

Reply:
@tanner0101 Done! :)